src/CalcManager/Ratpack/ratconst.h

```diff
@@ -1,480 +1,480 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-// Autogenerated by _dumprawrat in support.c
-NUMBER init_num_one= {
+// Autogenerated by _dumprawrat in support.cpp
+const NUMBER init_num_one= {
```

Comment:
Hey @jefgen, similar to the other PR, these should be updated to be `inline const` and for the same reasons.  Here's the size diff results:
- Release jefgen master: 4,353,536
- Release jefgen `const`:  4,352,512
- Release jefgen `inline const`: 4,352,000

So it looks like adding `inline` further reduces the number of duplicate items created.  In general, when defining const objects _in headers_, the right keyword is `inline` because the header will probably be included in multiple translation units and the linker will resolve to a single definition across the program.

Reply:
Interesting... I'm also able to replicate your results, and the size is indeed smaller when I add `inline` along with the `const`.

I was a bit puzzled why at first though, but taking a closer look at the linker output shows why:
It seems that many of the constants are not actually used anywhere in the program!
(For example, it seems that `init_num_two`, `init_num_ten`, etc. only exist in the header file).

Adding the `inline` allows the linker to simply drop these unused items, resulting in the smaller binary size.